packages/common/src/content.ts

```diff
@@ -138,3 +138,55 @@ export function getCollection(itemType: string = ""): string {
     }
   }
 }
+
+/**
+ * Case-insensitive check if the type is "Feature Service"
+ * @param {string} type - item's type
+ * @returns {boolean}
+ */
+export const isFeatureService = (type: string) => {
+  return type && type.toLowerCase() === "feature service";
+};
+
+/**
+ * parse layer id from a service URL
+ * @param {string} url
+ * @returns {string} layer id
+ */
+export const getLayerIdFromUrl = (url: string) => {
+  const endsWithNumberSegmentRegEx = /\/\d+$/;
+  const matched = url && url.match(endsWithNumberSegmentRegEx);
+  return matched && matched[0].slice(1);
+};
+
+/**
+ * return the layerId if we can tell that item is a single layer service
+ * @param {*} item from AGO
```

Comment:
Minor, comment formatting. Outside scope of this PR, but have we ever explored adding `eslint-plugin-jsdoc` or similar for enforcing consistent commenting/docing styles?

Reply:
so you are saying that the doc comments shouldn't have types, right? b/c that is taken care of by typedoc?

Reply:
I hadn't considered that. I guess it'd just be nice if our inline comments were aligned w/ the docs generated by typedoc.

Reply:
I'm not sure what's out of alignment, either specifically w/ this comment (other than I think it should not have the types), or in general, but if there are improvements w/ can make w/ `eslint-plugin-jsdoc` I say open an issue.